.eslintrc

```diff
@@ -0,0 +1,20 @@
+---
+extends: react-app
```

Comment:
@no-stack-dub-sack it may be smart to add rules as warnings first then errors once you get them fully resolved